src/api/spec/factories/upload_job.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :upload_job, class: Cloud::User::UploadJob do
     user
-    job_id { Faker::Number.between(100, 1000) }
+    job_id { rand(1_000_000_000) }
```

Comment:
can the `job_id` be 0? :thinking:  I think it should be fine...

Reply:
It can^^ Might not make much sense, but it wouldn't break anything